ScriptedEvents/Variables/Player/CustomRolePlayerVariable.cs.cs

```diff
@@ -0,0 +1,58 @@
+﻿namespace ScriptedEvents.Variables.Player.Roles
```

Comment:
Why is this file named `CustomRolePlayerVariable.cs.cs`? Rename to `CustomRolesVariables.cs`

Reply:
Oups Skill issue lol xD